packages/terra-clinical-result/src/proptypes/resulttypePropTypes.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+
+const resulttypePropOneOf = PropTypes.oneOf([
+  'ALPHA',
+  'MULTIALPHA',
+  'FREETEXT',
+  'NUMERIC',
+  'BLOODPRESSURE',
+  'CALCULATION',
+  'DATE',
+  'DATETIME',
+  'TIME',
+  'PROVIDER',
+]);
+
+export default resulttypePropOneOf;
+export {
+  resulttypePropOneOf,
```

Comment:
Do we need both exports?

Reply:
Removed extraneous export https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-97fd6d162e8139408020ff709b595847R16